lib/zendesk_apps_support/assets/templates.js.erb

```diff
@@ -0,0 +1,8 @@
+<% unless templates.nil? %>
+{
+  <% templates.each do |name, fn| %>
+  <%= name %>: <%= fn %>,
```

Comment:
I had to do the following to make it work:
That also what the normal compiler outputs btw (it wraps it into that Handlebars.template function).
```
-  <%= name %>: <%= fn %>,
+  <%= name %>: Handlebars.template(<%= fn %>),
```